edit-post/components/sidebar/post-status/index.js

```diff
@@ -35,6 +36,7 @@ function PostStatus( { isOpened, onTogglePanel } ) {
 		<PanelBody className="edit-post-post-status" title={ __( 'Status & Visibility' ) } opened={ isOpened } onToggle={ onTogglePanel }>
 			<PostVisibility />
 			<PostSchedule />
+			<AutosaveMessage />
```

Comment:
Might be worth some design feedback whether we want this text here, since the rest of the panel are options configurable by the user for publishing; this is just a text status, one which may not be important for the user at all.

Reply:
yes I would love some design feedback and we discussed this on the PR previously... @jasmussen suggested iterating once merged although I can't find the original comment.